LuaDisassembler.cs

```diff
@@ -2082,38 +2108,37 @@ public static void GenerateIRHKS(IR.Function irfun, LuaFile.Function fun)
             //irfun.PeepholeOptimize();
             irfun.CheckControlFlowIntegrity();
 
-            // Control flow graph construction and SSA conversion
+             // Control flow graph construction and SSA conversion
             irfun.ConstructControlFlowGraph();
             irfun.ResolveIndeterminantArguments(SymbolTable);
             irfun.CompleteLua51Loops();
             irfun.ConvertToSSA(SymbolTable.GetAllRegistersInScope());
 
-            // Data flow passes
+             // Data flow passes
             irfun.EliminateDeadAssignments(true);
             irfun.PerformExpressionPropogation();
             irfun.DetectListInitializers();
             irfun.PerformExpressionPropogation();
-
-            // CFG passes
+             // CFG passes
             irfun.StructureCompoundConditionals();
             irfun.DetectLoops();
             irfun.DetectLoopConditionals();
             irfun.DetectTwoWayConditionals();
             irfun.SimplifyIfElseFollowChain();
             irfun.EliminateDeadAssignments(true);
             irfun.PerformExpressionPropogation();
-            //irfun.VerifyLivenessNoInterference();
+            irfun.VerifyLivenessNoInterference();
```

Comment:
Leave this commented for now. There's some scripts this breaks on because it doesn't handle every case.